tests/buildenv.sh

```diff
@@ -1,2 +1,10 @@
-#!/usr/bin/env bash
+#!/usr/bin/sh
+
+set -eu
+
+python3 --version
+
+python3 -c "print('Dit is een {} test'.format('build env'))"
```

Comment:
:rofl: i don't think this was intended :)

i like the use of -c; maybe do the two modules we build as imports before the test :)